### PR TITLE
Initial commit for SC data release notes 19

### PR DESCRIPTION
**Pull request by pinin4fjords**
Starting release notes for March '22 SC release. The Jenkins job didn't work, so this is the result of me manually running the update script.

### CODE REVIEW

File: release-notes/sc/_posts/2022-03-14-19.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+- [Dissecting the cellular specificity of smoking effects and reconstructing lineages in the human airway epithelium (in vivo)](https://www.ebi.ac.uk/gxa/sc/experiments/E-CURD-114)
+- [Ischaemic sensitivity of human tissue by single cell RNA seq](https://www.ebi.ac.uk/gxa/sc/experiments/E-HCAD-1)
+- [Lung Cell Atlas: Single cell RNA sequencing analysis of fresh resected human lung tissue - Drop-seq dataset](https://www.ebi.ac.uk/gxa/sc/experiments/E-GEOD-130148)
+
+
+#### New experiments
+
+-Fly Cell Atlas datasets:
+- [Fly Cell Atlas: single-cell transcriptomes of the entire adult Drosophila - 10x dataset](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-10519)
+- - [Fly Cell Atlas: single-cell transcriptomes of the entire adult Drosophila - Smartseq2 dataset](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-10628)
```
Comment (pcm32) [inline release-notes/sc/_posts/2022-03-14-19.md]:
```suggestion
- [Fly Cell Atlas: single-cell transcriptomes of the entire adult Drosophila - Smartseq2 dataset](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-10628)
```

File: release-notes/sc/_posts/2022-03-14-19.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
+#### Data statistics
+
+- Ensembl **104** / Ensembl Genomes **51** / WormBase ParaSite **15** gene annotations.   
+- This release contains **304** single cell RNA-Seq studies, consisting of **13,826,582** cells, of which **8,524,149** passed our QC from **20** different species.
+
+#### New features
+
+- DiscovAIR collection
```
Comment (pcm32) [inline release-notes/sc/_posts/2022-03-14-19.md]:
```suggestion
```

File: release-notes/sc/_posts/2022-03-14-19.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+- New collection for experiments from the discovAIR project, it currently contains four experiments with more coming soon:
```
Comment (pcm32) [inline release-notes/sc/_posts/2022-03-14-19.md]:
```suggestion
- New collection for experiments from the [discovAIR](https://discovair.org/) project, it currently contains four experiments with more coming soon:
```

**Review by pcm32 — COMMENTED** [review]
Some minor suggestions

**Review by pcm32 — APPROVED** [review]
LGTM

**Comment by pinin4fjords** [comment]
Should we mention the annDatas?

File: release-notes/sc/_posts/2022-03-14-19.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+- [Human dendritic cell subsets in acute skin inflammation](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-8498)
+- [scRNAseq of Cyp11a1-expressing cells from mouse melanoma model derived tumors](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-8509)
+- [Single-cell RNA-seq of intestinal crypts of Red2Kras and Red2Pi3k mice](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-8656)
+- [Single cell RNA sequencing of chronic myelomonocytic leukemia stem cell](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-8884)
+- [Single-cell RNA-seq of the human fetal lateral ganglionic eminence at 7](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-8894)
+- [A spatially resolved single cell atlas of human gastrulation](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-9388)
+- [Single cell RNA sequencing of three IDHwt glioblastoma patient tumors](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-9435)
+- [Single cell transcriptional profile of third instar larval wing discs in Drosophila](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-9444)
+- [Single-cell RNA-seq on enriched cardiac fibroblasts from the healthy](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-9583)
```
Comment (anjaf) [inline release-notes/sc/_posts/2022-03-14-19.md]:
```suggestion
- [Single-cell RNA-seq on enriched cardiac fibroblasts from the healthy, sham or MI hearts of Hif-1a conditional knockout or heterozygous mice ](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-9583)
```

File: release-notes/sc/_posts/2022-03-14-19.md [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+- [Ischaemic sensitivity of human tissue by single cell RNA seq](https://www.ebi.ac.uk/gxa/sc/experiments/E-HCAD-1)
+- [Lung Cell Atlas: Single cell RNA sequencing analysis of fresh resected human lung tissue - Drop-seq dataset](https://www.ebi.ac.uk/gxa/sc/experiments/E-GEOD-130148)
+
+
+#### New experiments
+
+-Fly Cell Atlas datasets:
+- [Fly Cell Atlas: single-cell transcriptomes of the entire adult Drosophila - 10x dataset](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-10519)
+- [Fly Cell Atlas: single-cell transcriptomes of the entire adult Drosophila - Smartseq2 dataset](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-10628)
+
```
Comment (anjaf) [inline release-notes/sc/_posts/2022-03-14-19.md]:
```suggestion
Fly Cell Atlas datasets:
- [Fly Cell Atlas: single-cell transcriptomes of the entire adult Drosophila - 10x dataset](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-10519)
- [Fly Cell Atlas: single-cell transcriptomes of the entire adult Drosophila - Smartseq2 dataset](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-10628)

Other datasets:
```

**Comment by pcm32** [comment]
You mean the download of AnnData? Or a change in their contents? First is still not part of this release. Software release is configurable survey banner and DiscovAIR collection in the featured experiments.

**Comment by pinin4fjords** [comment]
> You mean the download of AnnData? Or a change in their contents? First is still not part of this release. Software release is configurable survey banner and DiscovAIR collection in the featured experiments.

well, I just meant that this is the first time we'll have the fully standardised layout in the objects for all datasets, as per https://github.com/ebi-gene-expression-group/scxa-workflows#anndata-format-output-files, even if just on FTP

**Comment by pcm32** [comment]
Sure, please add a couple of lines about it.

**Pull request merged**